Cargo.toml

```diff
@@ -17,36 +17,37 @@ default-target = "x86_64-unknown-linux-gnu"
 targets = ["i686-pc-windows-msvc", "x86_64-pc-windows-msvc", "i686-unknown-linux-gnu", "x86_64-unknown-linux-gnu", "x86_64-apple-darwin", "wasm32-unknown-unknown"]
 
 [features]
-default = ["x11", "wayland", "wayland-dlopen"]
+default = ["x11", "wayland", "wayland-dlopen", "kmsdrm"]
 x11 = ["x11-dl", "mio", "percent-encoding", "parking_lot"]
-wayland = ["wayland-client", "wayland-protocols", "sctk"]
+wayland = ["wayland-client", "wayland-protocols", "sctk", "calloop"]
 wayland-dlopen = ["sctk/dlopen", "wayland-client/dlopen"]
+kmsdrm = ["drm", "input", "calloop", "libseat", "xkbcommon", "udev"]
 
 [dependencies]
-instant = { version = "0.1", features = ["wasm-bindgen"] }
-lazy_static = "1"
-log = "0.4"
-serde = { version = "1", optional = true, features = ["serde_derive"] }
-raw-window-handle = "0.4.2"
-bitflags = "1"
-mint = { version = "0.5.6", optional = true }
+instant = { version = "0.1.12", features = ["wasm-bindgen"] }
+lazy_static = "1.4.0"
+log = "0.4.17"
+serde = { version = "1.0.137", optional = true, features = ["serde_derive"] }
+raw-window-handle = { git = "https://github.com/rust-windowing/raw-window-handle" }
```

Comment:
Well, the reason I'm using the Git dependency in this PR is because [I made a PR](https://github.com/rust-windowing/raw-window-handle/pull/91) to `raw-window-handle` which was merged but has not been released yet

Reply:
Same scenario with `xkbcommon`